src/main/java/ch/njol/unofficialmonumentamod/options/Options.java

```diff
@@ -1,174 +1,194 @@
-	@Category("effectOverlay")
-	@IntSlider(min = 50, max = 500)
-	public int effect_width = 200;
-
-	@Category("chestCountOverlay")
-	public boolean chestCount_enabled = true;
-	@Category("chestCountOverlay")
-	public ElementPosition chestCount_position = new ElementPosition(1f, 0, 0.0f, 0, 1f, 0);
-
-	@Category("debug")
-	public boolean debugOptionsEnabled = false;
-	@Category("debug")
-	public boolean logPackets = false;
-
-	public void onUpdate() {
-		if (abilitiesDisplay_preset != AbilityOptionPreset.CUSTOM) {
-			abilitiesDisplay_horizontal = abilitiesDisplay_preset.horizontal;
-			abilitiesDisplay_position = abilitiesDisplay_preset.position.clone();
-			abilitiesDisplay_preset = AbilityOptionPreset.CUSTOM;
-		}
-		UnofficialMonumentaModClient.saveConfig();
-	}
-
-	public boolean categoryVisible(String category) {
-		return debugOptionsEnabled || !category.equals("debug");
-	}
-
-}
+package ch.njol.unofficialmonumentamod.options;
+
+import ch.njol.minecraft.config.annotations.*;
+import ch.njol.minecraft.uiframework.ElementPosition;
+import ch.njol.unofficialmonumentamod.AbilityOptionPreset;
+import ch.njol.unofficialmonumentamod.UnofficialMonumentaModClient;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Options implements ch.njol.minecraft.config.Options {
+
+	@Category("misc")
+	public boolean overrideTridentRendering = true;
+	@Category("misc")
+	public boolean lowerVillagerHelmets = false;
+	@Category("misc")
+	public boolean hideVillagerPlayerHeads = false;
+
+	@Category("misc")
+	public boolean firmamentPingFix = true;
+
+	@Dropdown("chestsort")
+	@Category("misc")
+	public boolean chestsortDisabledForInventory = false;
+	@Dropdown("chestsort")
+	@Category("misc")
+	public boolean chestsortDisabledForEnderchest = false;
+	@Dropdown("chestsort")
+	@Category("misc")
+	public boolean chestsortDisabledEverywhereElse = false;
+
+	@Dropdown("location")
+	@Category("misc")
+	public boolean notifyLocation = false;
+	@Dropdown("location")
+	@Category("misc")
+	@FloatSlider(min = 1.5F, max = 20F, step = 0.1F, unit = " seconds")
+	public float notifierShowTime = 5F;
+
+	// TODO implement item cooldown display
+	// requires sever-side adaptions to send the cooldown (on use and on connect)
+
+	// the biggest issue: most tesseracts are apparently done in mcfunctions -> and one of them for some reason doesn't handle its charges the same way as the others.
+	// @Category("misc")
+	// public boolean renderItemCooldowns = true;
+
+	@Category("misc")
+	public boolean silenceTeamErrors = true;
+
+	@Dropdown("calculator")
+	@Category("misc")
+	public boolean showCalculator = true;
+	@Dropdown("calculator")
+	@Category("misc")
+	public boolean enableKeybindOutsidePlots = false;
+
+	@Category("misc")
+	public boolean enableTextureSpoofing = true;
+	
+	@Category("misc")
+	public transient DescriptionLine overlay_misc;
+	@Category("misc")
+	@FloatSlider(min = 0F, max = 1F, step = 0.05F, unit = "A/mm")
```

Comment:
What does "A/mm" mean here? Fells weird - shouldn't it be "%"?

Reply:
I remembered looking for the unit a while back and ending up with this. Possible it's wrong though, though since I'm not quite sure of it's validity I'll just replace it